src/Tribe/Template.php

```diff
@@ -647,14 +668,31 @@ public function template( $name, $context = array(), $echo = true ) {
 		 * Fires an Action after including the template file
 		 *
 		 * @since  4.6.2
-		 * @since  4.7.20   The $name param no longers contains the extension
+		 * @since  4.7.20   The $name param no longer contains the extension
 		 *
 		 * @param string $file      Complete path to include the PHP File
 		 * @param array  $name      Template name
 		 * @param self   $template  Current instance of the Tribe__Template
 		 */
 		do_action( 'tribe_template_after_include', $file, $name, $this );
 
+		/**
+		 * Fires an Action for a given template name after including the template file
+		 *
+		 * E.g.:
+		 *    `tribe_template_after_include:events/blocks/parts/details`
+		 *    `tribe_template_after_include:events/embed`
+		 *    `tribe_template_after_include:tickets/login-to-purchase`
+		 *
+		 * @deprecated 4.11.0
+		 * @since  4.7.20
+		 *
+		 * @param string $file      Complete path to include the PHP File
+		 * @param array  $name      Template name
+		 * @param self   $template  Current instance of the Tribe__Template
+		 */
+		do_action( "tribe_template_after_include:$legacy_hook_name", $file, $name, $this );
```

Comment:
:warning: Words in hook names should be separated using underscores. Expected: "tribe_template_after_include_$legacy_hook_name", but found: "tribe_template_after_include:$legacy_hook_name".

<sub>hash: 0cd4023f5066f41e29e60b3400b310c4</sub>